FirebasePerformance/Sources/FPRClient.m

```diff
@@ -179,15 +179,15 @@ - (void)logTrace:(FIRTrace *)trace {
 
       // Log the trace metric with its console URL.
       if ([trace.name hasPrefix:kFPRPrefixForScreenTraceName]) {
-        FPRLogDebug(kFPRClientMetricLogged,
+        FPRLogInfo(kFPRClientMetricLogged,
```

Comment:
I'm tempted to call this as `Debug` since these are helpful for the developers to analyze the issues in their application. From Perf SDK standpoint this might look as "Info", but since we have the console URLs now, it might make sense to call them debug. WDYT?

Reply:
This is the metric logging message "Logging trace metric - xxxx. In a minute, visit the Firebase console to view your data: xxx". Same as [the network request logging message](https://github.com/firebase/firebase-ios-sdk/blob/master/FirebasePerformance/Sources/FPRClient.m#L221-L223). Also, this is the purpose of this PR.

I would like to keep this as `Info`. Please let me know if you have any concerns.

Reply:
Discussed offline. Will log as `INFO` to let developer know a metric is logged.